core/record/_recordValidator/countValidator.js

```diff
@@ -47,63 +55,50 @@ const _getNodePointers = (survey, record, nodes) => {
     if (NodeDef.isEntity(nodeDef) && !Node.isDeleted(node)) {
       // Add children node pointers
       const childDefs = Survey.getNodeDefChildren(nodeDef)(survey)
-
-      for (const childDef of childDefs) {
-        nodePointers.push({
+      nodePointers.push(
+        ...childDefs.map((childDef) => ({
```

Comment:
is it always an array?

Reply:
childDefs is returned by Survey.getNodeDefChildren, and it's an array